src/cli/mod.rs

```diff
@@ -85,11 +85,15 @@ pub struct AddSubcommand {
     /// of installing it to the nearest aftman.toml file.
     #[structopt(long)]
     pub global: bool,
+
+    /// A Personal Access Token for private git repos
+    #[structopt(long)]
+    pub token: Option<String>
```

Comment:
I think we should start with only pulling authentication from a config file. If we add flags for authentication, we'll probably want to do that as a global flag instead of on each command.